docs/graphcache/errors.md

```diff
@@ -284,3 +284,27 @@ This error occurs when an unknown type is found in `opts.keys`.
 
 Check whether your schema is up-to-date, or whether you're using an invalid
 typename in `opts.keys`, maybe due to a typo.
+
+## (21) Invalid mutation <a id="21"></a>
+
+> Invalid mutation: `???` is not a mutation in the defined schema
```

Comment:
Small nit: The wording here is colloquially correct ("mutation" referring to part of the `Mutation` type, etc) but it may be more specific to say "Mutation field" here and "is not a Mutation field" / "is not a field on the Mutation type"

Reply:
Updated!